### PR TITLE
locust close connections for each request

### PR DESCRIPTION
**Pull request by annapendleton**
Locust user task uses requests library as underlying infra.

To ensure Locust loadbalances across new TGI pods with autoscaling, close the connection for each request so a new one is opened.

### CODE REVIEW

**Review by laoj2 — APPROVED** [review]

File: benchmarks/benchmark/tools/locust-load-inference/locust-docker/locust-tasks/tasks.py [hunk 1]
```diff
@@ -107,7 +107,7 @@ def lm_generate(self):
         prompt = test_data[random.randrange(0, len(test_data))]
 
         request = generate_request(model_params, prompt)
-        headers = {"User-Agent": "Benchmark Client"}
+        headers = {"User-Agent": "Benchmark Client", 'Connection':'close'}
```
Comment (laoj2) [inline benchmarks/benchmark/tools/locust-load-inference/locust-docker/locust-tasks/tasks.py:110]: can you use double quotation marks for consistency with the other strings?

Comment (annapendleton) [reply benchmarks/benchmark/tools/locust-load-inference/locust-docker/locust-tasks/tasks.py]: done!

**Pull request merged**